Assets/scripts/Enemy/Controller/ControllerBlueEnemy.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using Enemy.Controller;
+using UnityEngine;
+
+public class ControllerBlueEnemy
+{
+    public static implicit operator ControllerBlueEnemy(ControllerEnemy v)
```

Comment:
what's this implicit operator? 
and v? (rename it properly even if its for 1 line of code - remember serial killer) :)